### PR TITLE
Don't set run_service always to true (bsc#952144)

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

File: src/clients/ntp-client_proposal.rb [hunk 1]
```diff
@@ -352,7 +353,7 @@ def WriteNtpSettings(ntp_servers, ntp_server, run_service)
     def Write(param)
       ntp_servers = param["servers"] || []
       ntp_server = param["server"] || ""
-      run_service = param["run_service"] || true
+      run_service = param.fetch("run_service", true)
```
Comment (imobach) [inline src/clients/ntp-client_proposal.rb]: Good catch!

File: src/clients/ntp-client_proposal.rb [hunk 2]
```diff
@@ -345,6 +345,7 @@ def WriteNtpSettings(ntp_servers, ntp_server, run_service)
     #   server (taken from UI if empty)
     #   servers (intended to use all of opensuse.pool.ntp.org,
     # 	   but I did not have time to make it work)
+    #   run_service (set to true if empty)
```
Comment (kobliha) [inline src/clients/ntp-client_proposal.rb]: BTW, this still applies :)

**Comment by imobach** [comment]
LGTM

**Comment by kobliha** [comment]
LGTMtoo :) ;)

**Pull request merged**